analyzers/src/SonarAnalyzer.Common/SymbolicExecution/Roslyn/ProgramState.cs

```diff
@@ -125,10 +125,20 @@ public ProgramState ResetOperations()
 
     public IOperation ResolveCapture(IOperation operation) =>
         operation?.Kind == OperationKindEx.FlowCaptureReference
-        && this[IFlowCaptureReferenceOperationWrapper.FromOperation(operation).Id] is { } captured
+        && this[operation.ToFlowCaptureReference().Id] is { } captured
             ? captured
             : operation;
 
+    public IOperation ResolveCaptureAndUnwrapConversion(IOperation operation) =>
+        operation?.Kind switch
+        {
+            OperationKindEx.FlowCaptureReference => CaptureOperation.ContainsKey(operation.ToFlowCaptureReference().Id)
+                ? ResolveCaptureAndUnwrapConversion(ResolveCapture(operation))
+                : operation,
```

Comment:
Make this single line, the EOLs don't look very nice inside switch statements and this is short engouht not to violate the line length rule
```suggestion
            OperationKindEx.FlowCaptureReference => CaptureOperation.ContainsKey(operation.ToFlowCaptureReference().Id) ? ResolveCaptureAndUnwrapConversion(ResolveCapture(operation)) : operation,
```